bin/sapcli

```diff
@@ -76,8 +76,11 @@ def parse_command_line(argv):
     arg_parser.add_argument("--msserv", default=os.getenv("SAP_MSSERV"), help="Message server port")
     arg_parser.add_argument("--sysid", default=os.getenv("SAP_SYSID"), help="System ID (use if connecting via a "
                                                                             "message server")
-    arg_parser.add_argument("--group", default=os.getenv("SAP_GROUP"), help="Group (use if connecting via a message "
-                                                                            "server")
+    arg_parser.add_argument(
+        "--rfc_group",  # name "group" is already in use
+        default=os.getenv("SAP_GROUP"),
+        help="Group (use if connecting via a message "
+        "server")
```

Comment:
Can you keep the format of the core around, please?

Reply:
Done.